jump-core/src/main/java/com/bitdecay/jump/BitWorld.java

```diff
@@ -27,21 +28,24 @@
 	private int tileSize = 32;
 	private BitPointInt gridOffset = new BitPointInt(0, 0);
 	private BitBody[][] gridObjects = new BitBody[0][0];
-	private List<BitBody> bodies = new ArrayList<>();
+	private List<BitBody> dynamicBodies = new ArrayList<>();
+	private List<BitBody> kineticBodies = new ArrayList<>();
+	private List<BitBody> staticBodies = new ArrayList<>();
 
 	/**
 	 * A map of x to y to an occupying body.
 	 */
 	private Map<Integer, Map<Integer, Set<BitBody>>> occupiedSpaces;
 	private Map<BitBody, BitResolution> pendingResolutions;
+	private Map<BitBody, Set<BitBody>> contacts;
 
 	public static BitPoint gravity = new BitPoint(0, 0);
 	public static BitPoint maxSpeed = new BitPoint(2000, 2000);
```

Comment:
I had them static so that I didn't have to pass the world around. I hate when you have to pass one object all over the place just to have access to one variable on it -- It just feels bad, and it clutters up method signatures.

If we can come up with a good strategy to disseminate the gravity (that's the one that had the issues, mostly) I'm game to go about it a different way. I just didn't have any clean ideas other than pass it all over the place.